lib/ui.lib.php

```diff
@@ -713,7 +713,7 @@ function toggle_visible($element)
 function display_notification($message, $timeout = 5000)
 {
     echo "<script type='text/javascript'>";
-    echo "displayNotification('" . addslashes(json_encode($message)) . "', " . $timeout . ");";
+    echo "displayNotification('" . addslashes(json_encode($message, JSON_UNESCAPED_UNICODE)) . "', " . $timeout . ");";
```

Comment:
If you don't want to break php < 5.4, you can add something like:
`if defined('JSON_UNESCAPED_UNICODE')`